components/atom/popover/src/index.js

```diff
@@ -19,15 +18,20 @@ function AtomPopover({
   id,
   onClose = () => {},
   onOpen = () => {},
-  placement = PLACEMENTS.BOTTOM
+  placement = PLACEMENTS.BOTTOM,
+  showPopover
 }) {
-  targetRef = useRef()
-  const [showPopover, setShowPopover] = useState(false)
+  const targetRef = useRef()
+  const [internalShowPopover, setInternalShowPopover] = useState(showPopover)
```

Comment:
being it a boolean, why not naming it `showInternalPopover` instead?

Reply:
any other idea about the naming @jcger @midudev? Would you understand it if we call it `showInternalPopover`or do you prefer as it is `internalShowPopover` ?